src/components/confirm/confirm.spec.tsx

```diff
@@ -348,7 +336,9 @@ describe("Confirm", () => {
         .find('[data-element="confirm"]')
         .at(0);
 
-      expect(confirmButton.props().destructive).toBeTruthy();
+      expect(
+        (confirmButton as ReactWrapper<ButtonProps>).props().destructive
```

Comment:
you should rewrite the const instead of forcing a type here:
```
      const confirmButton = wrapper
        .find(Button)
        .filter('[data-element="confirm"]')
        .at(0);
```